comp/core/workloadmeta/collectors/internal/process/process_collector.go

```diff
@@ -0,0 +1,148 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package process implements the local process collector for
+// Workloadmeta.
```

Comment:
[minor] You can just make this a single line

Reply:
Done